drafts/use-cases/6.updating-event.md

```diff
@@ -0,0 +1,80 @@
+## Updating an existing event
+
+### Story
+As an application user
+I want to be able to update an existing events manually
+So I can have store some additional details bound to those events.
+
+As an API consumer
+I want to be able to update existing resources of a given type
+So I can amend content.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var event = client.get("/api/events/1")
+event.eventName = "Some new event name.";
+client.invoke(event.getOperations().filter(op => op.method == "PUT")[0], event);
```

Comment:
Related to discussed in https://github.com/HydraCG/Specifications/pull/117#discussion_r114505359

potentially http://schema.org/ReplaceAction could get used